### PR TITLE
cli: freva-plugin --unique_output it works properly

### PR DESCRIPTION
**Pull request by eelucio**
the behaviour is the same as the one from the python module

added a new dummyplugin to test that the folderpath has/has not appended the rowid

--

in that light I also rewrote the `freva-user-data index --continue_on_errors` to `freva-user-data index --abort_on_errors False` 
so the python and cli follow the same syntax

### CODE REVIEW

File: src/freva/_user_data.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def index(
         self,
         *crawl_dirs: os.PathLike,
         dtype: str = "fs",
-        continue_on_errors: bool = False,
+        abort_on_errors: bool = True,
```
Comment (antarcticrainforest) [inline src/freva/_user_data.py]: Why did you change that? I think it was better before?

File: src/freva/_user_data.py [hunk 2]
```diff
@@ -251,8 +251,8 @@ def index(
             The data path(s) that needs to be crawled.
         dtype:
             The data type, currently only files on the file system are supported.
-        continue_on_errors:
-            Continue indexing on error.
+        abort_on_errors:
```
Comment (antarcticrainforest) [inline src/freva/_user_data.py]: I think continue on errors was ok before

File: src/freva/cli/plugin.py [hunk 3]
```diff
@@ -92,9 +92,9 @@ def __init__(
         )
         self.parser.add_argument(
             "--unique_output",
-            help="Append a freva run id to every output folder",
-            default=True,
-            type=bool,
+            choices=["True", "False"],
+            help="Append a Freva run id to the output/cache folder(s).",
```
Comment (antarcticrainforest) [inline src/freva/cli/plugin.py:96]: If you wanted to do it like that then I would be inclined to say you make the choices lowercase.

Comment (antarcticrainforest) [reply src/freva/cli/plugin.py:96]: Also, could you do me a favour and call this `--unique-output`. I think back when we transitioned I stayed with `unique_output` because I wasn't to sure about the implications. I think now I can say it won't matter, except that `unique_output` doesn't look nice.

File: src/freva/cli/user_data.py [hunk 4]
```diff
@@ -35,11 +35,12 @@ def __init__(self, subparser: argparse.ArgumentParser):
             help="The data type of the data.",
         )
         self.parser.add_argument(
-            "--continue-on-errors",
-            "--continue",
-            "-c",
-            action="store_true",
-            help="Continue indexing on error.",
+            "--abort_on_errors",
```
Comment (antarcticrainforest) [inline src/freva/cli/user_data.py]: I like it much more as you did it before.

**Review by antarcticrainforest — CHANGES_REQUESTED** [review]

File: src/freva/_user_data.py (hunk 1 above)
Comment (eelucio) [reply src/freva/_user_data.py]: I changed after the talk we had in regards of being consistent between python and cli syntax, as this parameter is named `abort_on_errors` under python. But I can backpedal it without any problem.

File: src/freva/cli/plugin.py (hunk 3 above)
Comment (eelucio) [reply src/freva/cli/plugin.py:96]: I will do both. I will also check whether in the cli, there is any option that involves capitalisation and make it lowercase, just in case.

File: src/freva/_user_data.py (hunk 1 above)
Comment (antarcticrainforest) [reply src/freva/_user_data.py]: Ok, I thought that but, the `abort_on_erros` flag is passed to the  "internal" API. It's not really supposed to be used by the users, meaning anything that lives outside the `freva` namespace a.k.a `evaluation_system` can be seen as internal. Here we don't have to worry about consistency because it's not supposed to be exposed to the users. So it's totally fine to do what you did in the first place.

On the other hand with the unique output thingy, it's slightly different cos the cli would differ from the user `run_pulgin` method. And that's an inconsistency.  Does that make sense?

Comment (eelucio) [reply src/freva/_user_data.py]: thanks for the explanation, yes it does make sense. I will make the changes asap.

**Review by antarcticrainforest — APPROVED** [review]
LGTM!
